.eslintrc

```diff
@@ -26,5 +19,35 @@ rules:
   # enforce comma at eol (never before)
   comma-style: [2, "last"]
 
+  # 2 spaces indentation
+  indent: [2, 2]
+
+  # quotes are always double
+  quotes: [2, "double"]
+
+  # single empty lines only
+  no-multiple-empty-lines: [2, {"max": 1}]
+
   # allow functions declarations at bottom of scope
   no-use-before-define: [2, "nofunc"]
+
+  # semicolons are useless
+  semi: [2, "never"]
+
+  # consistent whitespace before function paren
+  space-before-function-paren: [2, {"anonymous": "always", "named": "never"}]
+
+  # consistent whitespace after keywords
+  space-after-keywords: [2, "always"]
+
+  # consistent whitespace in parens
+  space-in-parens: [2, "never"]
+
+  # consistent whitespace in brackets (disabled as buggy)
+  # space-in-brackets: [2, "always"]
+
```

Comment:
If buggy, please add as comment url of the related issue so we think to check/subscribe :)